### PR TITLE
Include check for bytes in extract_value, fixes #72

### LINKED ISSUE #72: add_bytes
I am trying to add a binary file to the index but having few issues

```python
    schema_builder = tantivy.SchemaBuilder()
    schema_builder.add_bytes_field("embedding")
    schema = schema_builder.build()

    tdoc = tantivy.Document(id=idx,
                                vector=np.frombuffer(emb, dtype=np.uint8).tolist(),
                            )
```
error:
```
 894/3555 [00:03<00:11, 235.92it/s]thread 'thrd-tantivy-index11' panicked at 'Bytes field contained non-Bytes Value!. Field Field(3) = FieldValue { field: Field(3), value: I64(162) }', /Users/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/tantivy-0.13.2/src/fastfield/bytes/writer.rs:57:21
 ```

Am I missing something?

### PR DESCRIPTION
**Pull request by cjrh**

### CODE REVIEW

**Comment by cjrh** [comment]
@ChillFish8 I'm attempting to fix #72, would appreciate your eyes to check if I'm wildly off or not. I'm still quite new to the tantivy codebase.

File: tests/tantivy_test.py [hunk 1]
```diff
@@ -531,3 +532,27 @@ def test_query_from_json_field(self):
...
+@pytest.mark.parametrize('bytes_kwarg', [True, False])
+@pytest.mark.parametrize('bytes_payload', [
+    b"abc",
+    bytearray(b"abc"),
+    memoryview(b"abc"),
+    BytesIO(b"abc").read(),
+    BytesIO(b"abc").getbuffer(),
+])
+def test_bytes(bytes_kwarg, bytes_payload):
+    schema = SchemaBuilder().add_bytes_field("embedding",).build()
```
Comment (ChillFish8) [inline tests/tantivy_test.py:546]: Nitpick, does this `,` here need to be there?

```suggestion
    schema = SchemaBuilder().add_bytes_field("embedding").build()
```

**Review by ChillFish8 — APPROVED** [review]
Looks good to me, small nitpick but nothing major 👍

File: tests/tantivy_test.py (hunk 1 above)
Comment (cjrh) [reply tests/tantivy_test.py:546]: oops, that made it over from the issue example

**Pull request merged**